python/mxnet/ndarray/numpy/_op.py

```diff
@@ -864,7 +925,7 @@ def sqrt(x, out=None, **kwargs):
 
     Parameters
     ----------
-    x : ndarray or scalar
+    x : ndarray
```

Comment:
scalar is supported

Reply:
it's no mine, i rebase from apache/numpy

Reply:
Be sure to update the numpy branch version before you push next time; the latest version has "scalar" added now.

Reply:
resolve